src/main/io/serial.c

```diff
@@ -352,42 +352,42 @@ serialPort_t *openSerialPort(
             break;
 #endif
 #ifdef USE_UART1
-        case SERIAL_PORT_USART1:
+        case SERIAL_PORT_UART1:
             serialPort = uartOpen(USART1, rxCallback, rxCallbackData, baudRate, mode, options);
             break;
 #endif
 #ifdef USE_UART2
-        case SERIAL_PORT_USART2:
+        case SERIAL_PORT_UART2:
             serialPort = uartOpen(USART2, rxCallback, rxCallbackData, baudRate, mode, options);
             break;
 #endif
 #ifdef USE_UART3
-        case SERIAL_PORT_USART3:
+        case SERIAL_PORT_UART3:
             serialPort = uartOpen(USART3, rxCallback, rxCallbackData, baudRate, mode, options);
             break;
 #endif
 #ifdef USE_UART4
-        case SERIAL_PORT_USART4:
+        case SERIAL_PORT_UART4:
             serialPort = uartOpen(UART4, rxCallback, rxCallbackData, baudRate, mode, options);
             break;
 #endif
 #ifdef USE_UART5
-        case SERIAL_PORT_USART5:
+        case SERIAL_PORT_UART5:
             serialPort = uartOpen(UART5, rxCallback, rxCallbackData, baudRate, mode, options);
             break;
 #endif
 #ifdef USE_UART6
-        case SERIAL_PORT_USART6:
+        case SERIAL_PORT_UART6:
             serialPort = uartOpen(USART6, rxCallback, rxCallbackData, baudRate, mode, options);
             break;
 #endif
 #ifdef USE_UART7
-        case SERIAL_PORT_USART7:
+        case SERIAL_PORT_UART7:
             serialPort = uartOpen(UART7, rxCallback, rxCallbackData, baudRate, mode, options);
             break;
 #endif
 #ifdef USE_UART8
-        case SERIAL_PORT_USART8:
+        case SERIAL_PORT_UART8:
             serialPort = uartOpen(UART8, rxCallback, rxCallbackData, baudRate, mode, options);
             break;
 #endif
```

Comment:
Could we get this cleaned up? It seems the `uartOpen()` call is always the same, just the port value is different.